Sources/HaCWebsiteLib/Events/Event.swift

```diff
@@ -0,0 +1,14 @@
+import Foundation
+
+protocol Event {
+    var title : String {get}
+    var time : Date {get}
+    var eventDescription: String {get} //NOT related to event content (e.g. workshop content)
+    var colour : String {get}
+    var hypeDays : Int {get}
+    var ttlDays : Int {get}
+    var imageURL : String? {get set}
```

Comment:
I see now why the variables in the implementations are mutable.

Why do these have to be mutable?